client/ui/qml/Pages2/PageHome.qml

```diff
@@ -242,6 +243,13 @@ PageType {
                 implicitHeight: parent.height - serversMenuHeader.implicitHeight
                 clip: true
 
+                function autoScroll(){
```

Comment:
Please add indents as in the rest of the project source files

Reply:
I think I understand the indents structure of the application now, please re-review, if there is a problem. please clarify it.